ui/scss/src/_dark-mode.scss

```diff
@@ -307,7 +307,7 @@ $dark-mode-input-border-focus: #007aff;
     .acn-parent-folder,
     .acn-admin-console,
     .ygtvcell {
-      color: $dark-mode-font-color;
+      color: $dark-mode-font-color !important;
```

Comment:
Can we increment the specificity of the relevant selector instead of !important?

Reply:
no, because we have a dark: a: color !important; and this is needed to override the default blue color of the links